packages/documentation-component/rollup.config.js

```diff
@@ -0,0 +1,68 @@
+import resolve from "rollup-plugin-node-resolve";
```

Comment:
Please create something like `rollup.base.js` with helpers in core of repository and then use functions from that file in rollup configs in packages.